### PR TITLE
Add TE link to new Activity Player

### PR DESCRIPTION
**Pull request by emcelroy**
https://www.pivotaltracker.com/story/show/175724569

[#175724569]

* Move preview options to select menu
* Add option to preview teacher edition in Activity Player

### CODE REVIEW

**Review by scytacki — APPROVED** [review]
This seems fine to me. It looks like it will conflict with the other work on selecting a runtime. So it should probably wait until that work is merged and this PR will need to be updated.

File: db/schema.rb
Comment (scytacki) [inline db/schema.rb:435]: Is this boolean supposed to be here? This PR doesn't seem to have a migration which adds it.

File: db/schema.rb
Comment (scytacki) [inline db/schema.rb:685]: Ditto from above.

File: app/helpers/lightweight_activity_helper.rb
Comment (scytacki) [inline app/helpers/lightweight_activity_helper.rb:64]: It'd be safer to use a URL utility here instead of just combining strings. Perhaps at some point the ACTIVITY_PLAYER_URL will have have its own parameters in it which would then break the code above.

**Review by scytacki — CHANGES_REQUESTED** [review]
I found a couple of things to update.

File: db/schema.rb
Comment (emcelroy) [reply db/schema.rb]: That snuck in from a different branch. I've removed it.

File: app/helpers/lightweight_activity_helper.rb
Comment (emcelroy) [reply app/helpers/lightweight_activity_helper.rb]: Sorry, I should've remembered that from your previous suggestion about using a URL utility elsewhere. Fixed.

**Review by scytacki — APPROVED** [review]
looks good. 👍

**Pull request merged**
